src/structify/resources/structure.py

```diff
@@ -259,6 +259,45 @@ def run_async(
             ),
             cast_to=object,
         )
+    
+    def run(
```

Comment:
We should probably have a docstring for this since it's public-facing

Reply:
Ack. Adding.